### PR TITLE
feat: enable smoke tests with dashmate

### PR DESCRIPTION
**Pull request by pshenmic**
## Issue being fixed or feature implemented

Make smoke tests work for nodes running dashmate

## What was done?

* Fix all tests according new infrastructure
* Enable HTTPS support for DAPI smoke tests
* Query via remote docker instead of querying service RPC ports (they are intentionally closed for security reasons)

## How Has This Been Tested?

Smoke tests on the running devnet

## Breaking Changes

No

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: ansible/group_vars/all [hunk 1]
```diff
@@ -52,7 +53,6 @@ tenderdash_log_level: debug
 dapi_image: dashpay/dapi
 dapi_envoy_image: envoyproxy/envoy
 dapi_port: 3000
-dapi_grpc_port: 3010
```
Comment (strophy) [inline ansible/group_vars/all:55]: Why is this removed?

File: karma.conf.js [hunk 2]
```diff
@@ -49,22 +63,23 @@ module.exports = (config) => {
       'karma-mocha',
       'karma-mocha-reporter',
       'karma-chai',
-      'karma-firefox-launcher',
```
Comment (strophy) [inline karma.conf.js:52]: Why are we changing this?

**Review by strophy — APPROVED** [review]

**Comment by pshenmic** [comment]
🤙🤙
